packages/components/src/presentation-components/properties/inputs/NumericPropertyInput.tsx

```diff
@@ -70,14 +56,25 @@ function getInputTargetFromPropertyRecord(propertyRecord: PropertyRecord) {
 }
 
 /** @internal */
-export interface NumericInputProps {
+export interface NumericInputProps extends PropertyEditorProps {
   onChange: (newValue: string) => void;
   onBlur?: React.FocusEventHandler;
   value: string;
+  isReadOnly?: boolean;
```

Comment:
I would rename to `isDisabled` or `disabld`.

Reply:
Renamed